services/filemanager/azureBlobStoragemanager.go

```diff
@@ -265,9 +313,11 @@ type AzureBlobStorageConfig struct {
 	Prefix         string
 	AccountName    string
 	AccountKey     string
+	SASToken       string
 	EndPoint       *string
 	ForcePathStyle *bool
 	DisableSSL     *bool
+	UseSASTokens   *bool
```

Comment:
To keep things simple, can we treat it as bool and treat the default value `False` as to be the base case ?

Reply:
The reason we started using *bool is to handle if the config key is not present. In that case pointer will points to nil.

Reply:
And we can handle that case as well.